metriq/client.py

```diff
@@ -359,6 +359,13 @@ def task_get_names(self) -> List[Task]:
             for r in response["data"]
         ]
 
+    @handler
+    def task_get_network(self) -> Task:
```

Comment:
This probably should not be here, right? This is specifically related to another issue and is not correct as implemented here.